frontend/packages/console-shared/src/components/form-utils/StickySection.tsx

```diff
@@ -0,0 +1,12 @@
+import * as React from 'react';
+import './StickySection.scss';
+
+const StickySection: React.FC = ({ children }) => {
```

Comment:
@karthikjeeyar We don't need a separate component for `StickySection`. We already have a shared `FormFooter` component in form utils which should be used in all the forms (It basically wraps the `ButtonBar` and `ActionGroup` components to provide consistent API for form footer) . I think we can make use of that component and simply add a `sticky: boolean` prop to that component. If the parent sends a `sticky` prop it will be sticky else not. What do you think?

Reply:
Sure, then we can make use of the FormFooter component everywhere instead of using `ButtonBar` and `ActionGroup` directly in all the Add flow forms

Reply:
yes , that makes sense